File: aoj/volume100/cpp/A10027.cpp
#include <iostream>

auto main() -> int {
	int a_score = 0;
	int b_score = 0;
	int n_competition;

	std::cin >> n_competition;
	for (int i = 0; i < n_competition; ++i) {
		std::string a_card;
		std::string b_card;
		std::cin >> a_card >> b_card;

		int result = a_card.compare(b_card);
		if (result == 0) {
			a_score += 1;
			b_score += 1;
		} else if (result > 0) {
			a_score += 3;
		} else if (result < 0) {
			b_score += 3;
		}
	}

	std::cout << a_score << " " << b_score << std::endl;
	return 0;
}

File: aoj/volume100/scala/A10027.scala
object Main {
  def main(args: Array[String]) {
    var a_score = 0
    var b_score = 0
    val n_compares = scala.io.StdIn.readInt
    for (_ <- 1 to n_compares) {
      val input = scala.io.StdIn.readLine split " "
      val a_card = input(0)
      val b_card = input(1)
      val result = a_card compare b_card
      if (result == 0) {
        a_score += 1
        b_score += 1
      } else if (result > 0) {
        a_score += 3
      } else if (result < 0) {
        b_score += 3
      }
    }
    println(s"$a_score $b_score")
  }
}
